test/Microsoft.Extensions.Logging.Test/ConsoleLoggerTest.cs

```diff
@@ -63,16 +63,19 @@ public void LogsWhenMessageIsNotProvided()
             var sink = t.Item2;
             var exception = new InvalidOperationException("Invalid value");
 
+            ((TestConsole)t.Item1.Console).OnFlush = () =>
```

Comment:
Another problem is that these Asserts don't cause the test to pass or fail since they run when the test method has finished,